### DISCUSSION TITLE
Macroeconomincal data

### DISCUSSION THREAD

**Discussion by datruccone** [alvarobartt/investpy#503] [Q&A]
Hi, is it possible to get historical macro data (GDP, Unemployment, Interest Rates, etc).
That info is present from the Economic Calendar BTW

Thanks!

**Comment by przemyslawbak** [comment]
Yes, it is:

```
import investpy
data = investpy.economic_calendar(
    from_date='12/09/2021',
    to_date  ='13/11/2021',
    countries=['germany'],
    time_zone = 'GMT',
)
print(data.head())
```

**Reply by datatalking** [reply]
I was going to ask a similar question, is there a master list of the data you referenced for the .investpy.economic_calendar? Where would that be?
